### DISCUSSION TITLE
Test fail on rocky Linux 9 HPC cluster

### DISCUSSION THREAD

**Discussion by AmbroiseJuston** [idaholab/moose#26570] [Q&A Getting Started] [answered]
Hello, i am a student building and testing moose on a HPC cluster running Rocky-9. I have installed moose onto a machine running a wsl on ubuntu before however following the same steps i seem to get 17 failed test (see below).

```pre
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test FAILED (ERRMSG)
fvkernels/mms/skewness-correction/adv-diff-react.average ...................................... FAILED (CRASH)
fvkernels/mms/skewness-correction/diffusion.average ........................................... FAILED (CRASH)
mortar/periodic-value.exo .................................................................... FAILED (ERRMSG)
fvkernels/mms/skewness-correction/adv-diff-react.skewcorrected ................................ FAILED (CRASH)
meshgenerators/file_mesh_generator.checkpoint_load_test ...................................... FAILED (ERRMSG)
meshgenerators/gmsh.gmsh_test ................................................................ FAILED (ERRMSG)
meshgenerators/gmsh_bcs.gmsh_bc_test ......................................................... FAILED (ERRMSG)
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/polycrystal_test . FAILED (ERRMSG)
fvkernels/mms/skewness-correction/diffusion.skewcorrected ..................................... FAILED (CRASH)
misc/ad_curvature.circle_tri6 ................................................................ FAILED (ERRMSG)
mortar/periodic-value.jac ................................................. FAILED (EXPECTED OUTPUT NOT FOUND)
restart/receiver.restart ..................................................................... FAILED (ERRMSG)
fvkernels/mms/skewness-correction/adv-diff-react.skewcorrected-advection ...................... FAILED (CRASH)
fvkernels/mms/skewness-correction/two_term_extrapol.skewcorrected ............................. FAILED (CRASH)
restart/restart_steady_from_transient.steady_from_transient_restart .......................... FAILED (ERRMSG)
meshgenerators/file_mesh_generator.checkpoint_old_prefix .................... FAILED (EXPECTED OUTPUT MISSING)
--------------------------------------------------------------------------------------------------------------
Ran 4139 tests in 127.6 seconds. Average test time 0.5 seconds, maximum test time 15.7 seconds.
4122 passed, 94 skipped, 0 pending, 17 FAILED
```

Any advice on what could be causing it or how to solve it would be appreiciated.
I did notice another user seemed to have problems with their mpich version and i double checked the provided solution. We currently have 4.1.1, would this be a noticeable difference in version that would cause these failure?

**Comment by milljm** [comment]
We need the more important information above these results in order to figure out why things actually failed.

The follow details the information we actually need:
https://mooseframework.inl.gov/help/troubleshooting.html#5e08e4ea-9123-4f2b-b148-f783ab616a5f

**Reply by AmbroiseJuston** [reply]
here is the full report for meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test for example:

```pre
./run_tests -i meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test 

meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: Working Directory: /home/justona-erau.edu/projects/moose/test/tests/meshgenerators/break_mesh_by_block_generator
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: Running command: /home/justona-erau.edu/projects/moose/test/moose_test-opt -i break_mesh_3D_polycrystal.i --mesh-only --error --error-override --no-gdb-backtrace
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: hwloc/linux: Ignoring PCI device with non-16bit domain.
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: Pass --enable-32bits-pci-domain to configure to support such devices
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: (warning: it would break the library ABI, don't enable unless really needed).
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: Exodus Library Warning/Error: [ex__check_file_type]
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 	ERROR: Could not recognize /home/justona-erau.edu/projects/moose/test/tests/meshgenerators/break_mesh_by_block_generator/poly2.msh as a valid Exodus/NetCDF file variant.  Magic value is '$Mes'
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: Exodus Library Warning/Error: [ex_open_int]
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 	ERROR: failed to open /home/justona-erau.edu/projects/moose/test/tests/meshgenerators/break_mesh_by_block_generator/poly2.msh of type 0 for reading. Either the file does not exist,
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 	or there is a permission or file format issue.
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 	NetCDF: Unknown file format
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: *** ERROR ***
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: Error opening ExodusII mesh file: /home/justona-erau.edu/projects/moose/test/tests/meshgenerators/break_mesh_by_block_generator/poly2.msh
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: Stack frames: 17
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 0: libMesh::print_trace(std::ostream&)
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 1: libMesh::MacroFunctions::report_error(char const*, int, char const*, char const*, std::ostream&)
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 2: libMesh::ExodusII_IO_Helper::open(char const*, bool)
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 3: libMesh::ExodusII_IO::read(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 4: FileMeshGenerator::generate()
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 5: MeshGenerator::generateInternal()
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 6: MeshGeneratorSystem::executeMeshGenerators()
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 7: Action::timedAct()
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 8: ActionWarehouse::executeActionsWithAction(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 9: ActionWarehouse::executeAllActions()
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 10: MooseApp::runInputFile()
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 11: MooseApp::run()
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 12: /home/justona-erau.edu/projects/moose/test/moose_test-opt(+0x20eb) [0x55729ad610eb]
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 13: main
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 14: /lib64/libc.so.6(+0x3feb0) [0x7f58fa43feb0]
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 15: __libc_start_main
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 16: /home/justona-erau.edu/projects/moose/test/moose_test-opt(+0x2396) [0x55729ad61396]
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: [0] ../src/mesh/exodusII_io_helper.C, line 683, compiled Nov 30 2023 at 16:46:29
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: Stack frames: 9
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 0: libMesh::print_trace(std::ostream&)
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 1: moose::internal::mooseErrorRaw(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 2: void mooseError<char const*>(char const*&&)
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 3: /home/justona-erau.edu/projects/moose/framework/libmoose-opt.so.0(+0x9e80bd) [0x7f59037e80bd]
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 4: /home/justona-erau.edu/projects/moose/test/moose_test-opt(+0x20eb) [0x55729ad610eb]
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 5: main
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 6: /lib64/libc.so.6(+0x3feb0) [0x7f58fa43feb0]
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 7: __libc_start_main
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 8: /home/justona-erau.edu/projects/moose/test/moose_test-opt(+0x2396) [0x55729ad61396]
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: [unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: :
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: system msg for write_line failure : Bad file descriptor
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: ################################################################################
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: Tester failed, reason: ERRMSG
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test: 
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test FAILED (ERRMSG)


Final Test Results:
--------------------------------------------------------------------------------------------------------------
meshgenerators/break_mesh_by_block_generator.break_mesh_by_blocks_generator/3d_polycrystal_test FAILED (ERRMSG)
```

Let me know if this format is acceptable.

**Reply by milljm** [reply]
@roystgnr Any ideas?

It _could_ be MPICH 4.1.1. I know I have had issues with versions greater than 4.0.3 lately. But I don't recall the specific error message.

**Reply by GiudGiud** [reply]
This test in particular I dont know.

The MMS tests in the list definitely point to MPI, they tend to be run with MPI

**Reply by milljm** [reply]
@AmbroiseJuston Can you paste a few more details from the other tests?

**Reply by roystgnr** [reply, marked as answer]
The poly2.msh failure looks as if we somehow instructed an `ExodusII_IO` object to open a non-ExodusII (looks like GMSH?) mesh.

But the way we decide to use `ExodusII_IO` is a *really* simple search through ...

Oh, no.

You're in a directory named `justona-erau.edu`.  Our *stupidly* simple search sees ".e" (and doesn't see ".ele") and it concludes you've got an Exodus file.

We need to fix that.  We can't only count files that end with ".e", because lots of users end up with files that have timestep numbers or such appended past that, but we can at least make sure that there aren't any directory separator characters in the file name after the "suffix".

In the meantime ... does your cluster give you a /tmp or /scratch or some other such directory you can use where you can create subdirectories without ".e" in them?  You wouldn't even have to create a real subdirectory, just a symlink back to your home directory, if that's necessary for reasons of speed or backup policy or whatever.

**Reply by GiudGiud** [reply]
Ah good catch.

And now that you mention it, those mms tests use a .msh file for the mesh. Because that s how we generated the skewed meshes. 

@AmbroiseJuston please rename the folder as Roy says, and we ll fix the problem in libmesh when we can.

**Reply by AmbroiseJuston** [reply]
Thank you very much @GiudGiud @roystgnr @milljm for the help, i'll try to do that and see if it fixes the other error too. I'll keep you guys updated if it returns anything more unusual that i can't fix myself.

@milljm here is other information from the second failed test like you asked:

Ran 1 tests in 3.1 seconds. Average test time 0.2 seconds, maximum test time 0.2 seconds.
(moose) [justona@lprc-b72-112-fem test]$ ./run_tests --re=fvkernels/mms/skewness-correction/adv-diff-react.average
fvkernels/mms/skewness-correction/adv-diff-react.average: Working Directory: /home/justona-erau.edu/projects/moose/test/tests/fvkernels/mms/skewness-correction/adv-diff-react
fvkernels/mms/skewness-correction/adv-diff-react.average: Running command: python3 -m unittest -v test.TestAverageStencil
fvkernels/mms/skewness-correction/adv-diff-react.average: Running: /home/justona-erau.edu/projects/moose/test/moose_test-opt -i skewed.i Variables/v/face_interp_method=average Mesh/uniform_refine=0
fvkernels/mms/skewness-correction/adv-diff-react.average: test (test.TestAverageStencil) ... hwloc/linux: Ignoring PCI device with non-16bit domain.
fvkernels/mms/skewness-correction/adv-diff-react.average: Pass --enable-32bits-pci-domain to configure to support such devices
fvkernels/mms/skewness-correction/adv-diff-react.average: (warning: it would break the library ABI, don't enable unless really needed).
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: Exodus Library Warning/Error: [ex__check_file_type]
fvkernels/mms/skewness-correction/adv-diff-react.average: 	ERROR: Could not recognize /home/justona-erau.edu/projects/moose/test/tests/fvkernels/mms/skewness-correction/adv-diff-react/skewed.msh as a valid Exodus/NetCDF file variant.  Magic value is '$Mes'
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: Exodus Library Warning/Error: [ex_open_int]
fvkernels/mms/skewness-correction/adv-diff-react.average: 	ERROR: failed to open /home/justona-erau.edu/projects/moose/test/tests/fvkernels/mms/skewness-correction/adv-diff-react/skewed.msh of type 0 for reading. Either the file does not exist,
fvkernels/mms/skewness-correction/adv-diff-react.average: 	or there is a permission or file format issue.
fvkernels/mms/skewness-correction/adv-diff-react.average: 	NetCDF: Unknown file format
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: *** ERROR ***
fvkernels/mms/skewness-correction/adv-diff-react.average: Error opening ExodusII mesh file: /home/justona-erau.edu/projects/moose/test/tests/fvkernels/mms/skewness-correction/adv-diff-react/skewed.msh
fvkernels/mms/skewness-correction/adv-diff-react.average: Stack frames: 17
fvkernels/mms/skewness-correction/adv-diff-react.average: 0: libMesh::print_trace(std::ostream&)
fvkernels/mms/skewness-correction/adv-diff-react.average: 1: libMesh::MacroFunctions::report_error(char const*, int, char const*, char const*, std::ostream&)
fvkernels/mms/skewness-correction/adv-diff-react.average: 2: libMesh::ExodusII_IO_Helper::open(char const*, bool)
fvkernels/mms/skewness-correction/adv-diff-react.average: 3: libMesh::ExodusII_IO::read(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
fvkernels/mms/skewness-correction/adv-diff-react.average: 4: FileMeshGenerator::generate()
fvkernels/mms/skewness-correction/adv-diff-react.average: 5: MeshGenerator::generateInternal()
fvkernels/mms/skewness-correction/adv-diff-react.average: 6: MeshGeneratorSystem::executeMeshGenerators()
fvkernels/mms/skewness-correction/adv-diff-react.average: 7: Action::timedAct()
fvkernels/mms/skewness-correction/adv-diff-react.average: 8: ActionWarehouse::executeActionsWithAction(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)
fvkernels/mms/skewness-correction/adv-diff-react.average: 9: ActionWarehouse::executeAllActions()
fvkernels/mms/skewness-correction/adv-diff-react.average: 10: MooseApp::runInputFile()
fvkernels/mms/skewness-correction/adv-diff-react.average: 11: MooseApp::run()
fvkernels/mms/skewness-correction/adv-diff-react.average: 12: /home/justona-erau.edu/projects/moose/test/moose_test-opt(+0x20eb) [0x5648131290eb]
fvkernels/mms/skewness-correction/adv-diff-react.average: 13: main
fvkernels/mms/skewness-correction/adv-diff-react.average: 14: /lib64/libc.so.6(+0x3feb0) [0x7ffb8703feb0]
fvkernels/mms/skewness-correction/adv-diff-react.average: 15: __libc_start_main
fvkernels/mms/skewness-correction/adv-diff-react.average: 16: /home/justona-erau.edu/projects/moose/test/moose_test-opt(+0x2396) [0x564813129396]
fvkernels/mms/skewness-correction/adv-diff-react.average: [0] ../src/mesh/exodusII_io_helper.C, line 683, compiled Nov 30 2023 at 16:46:29
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: Stack frames: 9
fvkernels/mms/skewness-correction/adv-diff-react.average: 0: libMesh::print_trace(std::ostream&)
fvkernels/mms/skewness-correction/adv-diff-react.average: 1: moose::internal::mooseErrorRaw(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)
fvkernels/mms/skewness-correction/adv-diff-react.average: 2: void mooseError<char const*>(char const*&&)
fvkernels/mms/skewness-correction/adv-diff-react.average: 3: /home/justona-erau.edu/projects/moose/framework/libmoose-opt.so.0(+0x9e80bd) [0x7ffb901e80bd]
fvkernels/mms/skewness-correction/adv-diff-react.average: 4: /home/justona-erau.edu/projects/moose/test/moose_test-opt(+0x20eb) [0x5648131290eb]
fvkernels/mms/skewness-correction/adv-diff-react.average: 5: main
fvkernels/mms/skewness-correction/adv-diff-react.average: 6: /lib64/libc.so.6(+0x3feb0) [0x7ffb8703feb0]
fvkernels/mms/skewness-correction/adv-diff-react.average: 7: __libc_start_main
fvkernels/mms/skewness-correction/adv-diff-react.average: 8: /home/justona-erau.edu/projects/moose/test/moose_test-opt(+0x2396) [0x564813129396]
fvkernels/mms/skewness-correction/adv-diff-react.average: application called MPI_Abort(MPI_COMM_WORLD, 1) - process 0
fvkernels/mms/skewness-correction/adv-diff-react.average: [unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=1
fvkernels/mms/skewness-correction/adv-diff-react.average: :
fvkernels/mms/skewness-correction/adv-diff-react.average: system msg for write_line failure : Bad file descriptor
fvkernels/mms/skewness-correction/adv-diff-react.average: ERROR
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: ======================================================================
fvkernels/mms/skewness-correction/adv-diff-react.average: ERROR: test (test.TestAverageStencil)
fvkernels/mms/skewness-correction/adv-diff-react.average: ----------------------------------------------------------------------
fvkernels/mms/skewness-correction/adv-diff-react.average: Traceback (most recent call last):
fvkernels/mms/skewness-correction/adv-diff-react.average:   File "/home/justona-erau.edu/projects/moose/test/tests/fvkernels/mms/skewness-correction/adv-diff-react/test.py", line 7, in test
fvkernels/mms/skewness-correction/adv-diff-react.average:     df1 = mms.run_spatial('skewed.i', 5, 'Variables/v/face_interp_method=average')
fvkernels/mms/skewness-correction/adv-diff-react.average:   File "/home/justona-erau.edu/projects/moose/python/mms/runner.py", line 133, in run_spatial
fvkernels/mms/skewness-correction/adv-diff-react.average:     return _runner(*args, rtype=SPATIAL, **kwargs)
fvkernels/mms/skewness-correction/adv-diff-react.average:   File "/home/justona-erau.edu/projects/moose/python/mms/runner.py", line 106, in _runner
fvkernels/mms/skewness-correction/adv-diff-react.average:     raise IOError("The CSV output does not exist: {}".format(csv))
fvkernels/mms/skewness-correction/adv-diff-react.average: OSError: The CSV output does not exist: None
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: ----------------------------------------------------------------------
fvkernels/mms/skewness-correction/adv-diff-react.average: Ran 1 test in 0.155s
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: FAILED (errors=1)
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average: Exit Code: 1
fvkernels/mms/skewness-correction/adv-diff-react.average: ################################################################################
fvkernels/mms/skewness-correction/adv-diff-react.average: Tester failed, reason: CRASH
fvkernels/mms/skewness-correction/adv-diff-react.average: 
fvkernels/mms/skewness-correction/adv-diff-react.average ...................................... FAILED (CRASH)


Final Test Results:
--------------------------------------------------------------------------------------------------------------
fvkernels/mms/skewness-correction/adv-diff-react.average ...................................... FAILED (CRASH)
--------------------------------------------------------------------------------------------------------------
Ran 1 tests in 4.1 seconds. Average test time 1.0 seconds, maximum test time 1.0 seconds.
0 passed, 0 skipped, 0 pending, 1 FAILED

**Reply by GiudGiud** [reply]
> @milljm here is other information from the second failed test like you asked:

second test shows the same failure.
The failure is in the mesh reader in libmesh.

The only thing you can do is rename your folder to not include `.e` in the name anywhere

**Reply by AmbroiseJuston** [reply]
I worked it out and renamed the folder and the tests now all pass. It indeed simply was that it had .e somewhere.
